### ISSUE TITLE
Break Out HTMLTemplateElement

### ISSUE THREAD

**Issue by EisenbergEffect** [webcomponents/webcomponentsjs#45]
I see the list of Web Components specs listed in the readme, but HTMLTemlateElement isn't listed for some reason. Does this library include it? If so, where is that? Also, would it be possible to break that out into an individual file like the other polyfills?

**Comment by curioustechizen** [comment]
I see that change [proposed above](https://github.com/webcomponents/webcomponentsjs/issues/45#issuecomment-63403215) is already part of the released code. However, I am having problems using it without also including HTMLImports polyfill in IE.

Basically, I tried to create a version of webcomponentsjs that only includes Custom Elements and Templates (no Shadow DOM, no HTML Imports). The reasoning is that if I use vulcanize to squash all imports into a single HTML page + single JS file, then HTML imports are not necessary. The resulting `webcomponents-feather.min.js` that I created is ~16KB (~5.6KB gzipped) which is a saving of around 3.5KB over webcomponents.min.js (gzipped).

When I do this, on IE10 I see that the `HTMLTemplateElement`s do not get bootstrapped when I am in the `createdCallback()` life cycle of my custom elements. IE cannot find the `.content` property on the `HTMLTemplateElement`. Using a debugger reveals that IE still sees the element as an `HTMLUnknownElement`. However, things work fine when I am in the `attachedCallback()` life cycle method.

Any ideas on how I can get around this? Perhaps there is an event my custom element can listen for before attempting to reach into the template content? I suppose `DOMContentLoaded` is too soon and `WebComponentsReady` is too late?
